app/views/reservations/_new_request.html.erb

```diff
@@ -33,12 +38,35 @@
   <div class="well">
     <%= markdown(@request_text) %>
   </div>
-  <div class="form-group text <%= 'error' if @notes_required %>">
-    <%= f.label "Enter your comments here:" %>
-    <%= f.input_field :notes, class: 'form-control', rows: 5 %>
-  </div>
-  <div class="form-group">
-    <%= f.button :submit, "Submit Request", data: { confirm: "Are you sure? This request is not finalized and will be subject to approval by an administrator. It will not be available for checkout until and unless approved, and the request may be denied for any reason at the sole discretion of the administrator." } %>
-    <%= link_to "Return to Catalog", catalog_path, class: 'btn btn-default' %>
+
+  <%= simple_form_for @reservation do |f| %>
+    <div class="form-group text <%= 'error' if @notes_required %>">
+      <%= f.label "Enter your comments here:" %>
+      <%= f.input_field :notes, class: 'form-control', rows: 5 %>
+    </div>
+    <div class="form-group">
+      <%= f.button :submit, "Submit Request", data: { confirm: "Are you sure? This request is not finalized and will be subject to approval by an administrator. It will not be available for checkout until and unless approved, and the request may be denied for any reason at the sole discretion of the administrator." } %>
+      <%= link_to "Return to Catalog", catalog_path, class: 'btn btn-default' %>
+    </div>
+  <% end %>
+</div>
+<%# edit reservation here%>
+<div id="form" class="well hidden">
```

Comment:
also, the selector name is a bit too general, it should probably be something like `edit_reservation_form`.